lib/take_away.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'order'
+
+class TakeAway
+  attr_reader :selection, :current_amount, :menu
+
+  def initialize
+    @menu = {
```

Comment:
Setting the menu in this way isn't necessarily a problem, but it does make it more difficult to alter - if the takeaway wants to change it's menu, or if you want multiple restaurants which sell different items to use your app it could be an issue.

If you were to create a separate menu class and hold this hash of items in there,  you could inject it in the initialiser and make things a bit more flexible.

Something like:

def initialize(menu = Menu.new) 
     @menu = menu
end